scripts/database/upgrades/upgrade_v4.15.1_to.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE ONLY setting DROP CONSTRAINT setting_pkey ;
+
+ALTER TABLE setting ADD COLUMN ID SERIAL PRIMARY KEY;
```

Comment:
does this add values for the id for existing settings? (it must, right?)

Also, can you update the file name to reference 4.16 (the latest release?)

Reply:
Yes, it adds serial number to all existing rows.  I will rename the file and commit it.

Reply:
committed the changes.